tensorflow/examples/label_image/main.cc

```diff
@@ -93,7 +93,8 @@ Status ReadTensorFromImageFile(string file_name, const int input_height,
 
   string input_name = "file_reader";
   string output_name = "normalized";
-  auto file_reader = ReadFile(root.WithOpName(input_name), file_name);
+  auto file_reader = tensorflow::ops::ReadFile(root.WithOpName(input_name),
```

Comment:
Out of curiosity, shouldn't the `using namespace ::tensorflow::ops;` on L92 make this change unnecessary?